pkg/services/ngalert/store/error/image.go

```diff
@@ -0,0 +1,11 @@
+package error
+
+import (
+	"errors"
+)
+
+var (
+	// ErrImageNotFound is returned when the image does not exist.
+	ErrImageNotFound     = errors.New("image not found")
```

Comment:
I put these in the `models` package because of this:
> proposal: we should consolidate our error constants into a package that doesn’t import anything else. I’ve run into a few import cycles recently that required making small local interfaces to avoid.
I actually like small, local interfaces a lot, but if I’m checking results against error constants, I can’t use any of the constants from the package I’m avoiding. If our errors were in the models package, or an ngalert/errs package we could check errors.Is without getting fiddly about it.

I'd prefer these to remain in either a 'models' package or an 'ngalert/error' package rather than being store-specific.

Reply:
I'm not sure I agree with the proposal as I think errors should be local to the package that returns them. The motivation for moving them out is that those errors are unrelated to the models as these are errors for the store (i.e. if we remove the store the errors no longer make sense). I will put them back in the models package for the time being though!

Reply:
There's certainly a judgement call involved - one thing I like about a shared "service level" errors package is that you don't do error translation between packages in a hierarchy:

```go
if errors.Is(err, store.ErrorInternal) {
  return ngalert.ErrorInternal(err) // This is some fmt.Errorf wrapper.
}
```

but this is like ... not the biggest deal.